analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UnusedPrivateMember.Fixed.Batch.cs

```diff
@@ -95,6 +95,14 @@ private interface MyInterface
         {
             void Method();
         }
+
+        public void MethodUsingLocalMethod()
+        {
+            void LocalMethod() // Compliant - FN: local function is never used
```

Comment:
nitpick: better to not use Compliant for FN - better just leave `// FN: local function is never used`

(we use "Compliant" when it would be a FP to raise)

Reply:
Noted. Removed "Compliant" from the comment.